cmake/dependencies/mosquitto.cmake

```diff
@@ -5,7 +5,10 @@
     #pkg_check_modules(Mosquitto IMPORTED_TARGET libmosquittopp REQUIRED)
 #endif()
 
-file (DOWNLOAD https://test.mosquitto.org/ssl/mosquitto.org.crt mosquitto.org.crt)
+file (DOWNLOAD
+   https://test.mosquitto.org/ssl/mosquitto.org.crt
+   ${CMAKE_BINARY_DIR}/demo/mosquitto.org.crt
```

Comment:
Sadly, this won't work with multi-config CMake generators (like "Ninja Multi-Config" or Visual Studio). With multi config, the demo binary goes to `${CMAKE_BINARY_DIR}/demo/Release/mecaps_demo_ui` but the cert goes to `${CMAKE_BINARY_DIR}/demo`.

I'm not sure how to deal with this exactly but probably something similar to what Slint does with copying .dlls on windows has to be done.